dashboard/app/bisect_test.go

```diff
@@ -1138,8 +1076,8 @@ func TestBugBisectionResults(t *testing.T) {
 	c.expectTrue(bytes.Contains(content, []byte("kernel: add a fix")))
 }
 
-// Test that bisection status shows up on main page.
-func TestBugBisectionStatus(t *testing.T) {
```

Comment:
The old test seems to be replaced by another one. Are we still checking what `TestBugBisectionStatus` did?